CorsixTH/Lua/dialogs/bottom_panel.lua

```diff
@@ -218,10 +218,21 @@ function UIBottomPanel:drawReputationMeter(canvas, x_left, y)
   self.panel_sprites:draw(canvas, 36, x_left + math.floor(step * (self.ui.hospital.reputation - self.ui.hospital.reputation_min)), y)
 end
 
+--! Adds dynamic text to the bottom panel based on cursor position
+--!param canvas
+--!param x (num) coordinate
+--!param y (num) coordinate
 function UIBottomPanel:drawDynamicInfo(canvas, x, y)
```

Comment:
This line and line 221 are now no longer fully correct, you also draw "paused" in some cases.

Reply:
Guess you're right, Maybe just a line in the comments above probably would have sufficed.